### PR TITLE
Driver and Modules Version Update

### PR DESCRIPTION
**Pull request by rishabhatdell**
#### Is this a new chart?

No

#### What this PR does / why we need it:

This PR upgrades the driver and module versions

#### Which issue(s) is this PR associated with:

- https://github.com/dell/csm/issues/1091

#### Checklist:

[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]

- [x] Chart Version bumped
- [ ] Variables are documented in the chart README.md
- [ ] Title of the PR starts with the chart name (e.g. `[charts_dir/mychartname]`) if applicable

### CODE REVIEW

File: charts/container-storage-modules/Chart.yaml
Comment (AkshaySainiDell) [inline charts/container-storage-modules/Chart.yaml:1]: Module version update is missing in this file.

**Review by AkshaySainiDell — CHANGES_REQUESTED** [review]

**Review by shanmydell — CHANGES_REQUESTED** [review]
Are observability,authorization,topology repos going to have releases? @rishabhatdell

**Comment by shanmydell** [comment]
@rishabhatdell : Those changes will not impact modules much, awaiting confirmation, will update the comment based on confirmation

File: charts/container-storage-modules/Chart.yaml
Comment (KshitijaKakde) [reply charts/container-storage-modules/Chart.yaml:1]: Updated the module version

**Comment by rishabhatdell** [comment]
@shanmydell any updates on modules?

**Review by shanmydell — APPROVED** [review]

**Review by AkshaySainiDell — APPROVED** [review]

**Pull request merged**